web_ui/src/components/ActivityPage.tsx

```diff
@@ -9,6 +9,32 @@ import { Current } from "../world"
 import { useTeamApi } from "../useApi"
 import { IActiveMergeQueue } from "../api"
 
+function useRenderOnInterval(seconds: number) {
```

Comment:
this seems weird

Essentially we want the `formatDistanceToNowStrict` to update on an interval, so maybe that should be a hook that takes a date and returns a string that updates every so often